libs/ledger-live-common/src/families/stellar/api/horizon.ts

```diff
@@ -276,41 +290,37 @@ export const buildPaymentOperation = ({
   amount: BigNumber;
   assetCode: string | undefined;
   assetIssuer: string | undefined;
-}): any => {
+}) => {
   const formattedAmount = getFormattedAmount(amount);
   // Non-native assets should always have asset code and asset issuer. If an
   // asset doesn't have both, we assume it is native asset.
-  const asset =
-    assetCode && assetIssuer
-      ? new StellarSdk.Asset(assetCode, assetIssuer)
-      : StellarSdk.Asset.native();
-  return StellarSdk.Operation.payment({
+  const asset = assetCode && assetIssuer ? new Asset(assetCode, assetIssuer) : Asset.native();
+  return StellarOperation.payment({
     destination: destination,
     amount: formattedAmount,
     asset,
-    withMuxing: true,
```

Comment:
Why that change?

Reply:
`withMuxing` is not a valid argument: https://stellar.github.io/js-stellar-sdk/Operation.html#.payment